README.rst

```diff
@@ -3,13 +3,13 @@ unasync
 
 |documentation| |travis| |appveyor| |codecov|
 
-.. |travis| image:: https://travis-ci.com/RatanShreshtha/unasync.svg?branch=master
+.. |travis| image:: .. image:: https://travis-ci.com/python-trio/unasync.svg?branch=master
```

Comment:
Can you please remove the two dots in a new PR? The image is now broken because of this, I think. Sorry that I missed it!